src/engine/settings.cpp

```diff
@@ -74,6 +74,11 @@ namespace version1 {
         }
         settings.bypassLauncher = get_to<bool>(json, "bypass");
 
+        std::optional<std::string> layerServer = get_to<std::string>(json, "layerServer");
```

Comment:
```suggestion
        std::optional<std::string> layerServer = get_to<std::string>(json, "layerserver");
```

Since we do lowercase characters for the keys in JSON objects